# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParent.h
# Commit: 37b7616c8b42
# Full Hash: 37b7616c8b4288e6f92e99ca1314e1f0169d7ed6
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 2
# Description:
#   Bug 1786465 - Convert FileSystemDataManager to be ref counted; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155551
# ==============================================================================

diff -r 399dce373c59 -r 37b7616c8b42 dom/fs/parent/FileSystemManagerParent.h
--- a/dom/fs/parent/FileSystemManagerParent.h	Wed Aug 31 09:34:54 2022 +0000
+++ b/dom/fs/parent/FileSystemManagerParent.h	Wed Aug 31 09:34:54 2022 +0000
@@ -75,7 +75,7 @@
  private:
   RefPtr<TaskQueue> mTaskQueue;
 
-  UniquePtr<fs::data::FileSystemDataManager> mDataManager;
+  RefPtr<fs::data::FileSystemDataManager> mDataManager;
 
   const EntryId mRootEntry;
 };